pkg/resource/processing_job/testdata/sdkapi/describe/stopping_describe.json

```diff
@@ -0,0 +1,103 @@
+    "AppSpecification": {
+        "ContainerArguments": null,
+        "ContainerEntrypoint": [
+            "python",
+            "/opt/ml/processing/code/kmeans_preprocessing.py"
+        ],
+        "ImageUri": "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.5.0-cpu-py36-ubuntu16.04/pytorch-training:1.5.0-cpu-py36-ubuntu16.04"
+    },
+    "AutoMLJobArn": null,
+    "CreationTime": "2021-09-29T05:51:48.716Z",
+    "Environment": {
+        "my_var": "my_value",
+        "my_var2": "my_value2"
+    },
+    "ExitMessage": null,
+    "ExperimentConfig": null,
+    "FailureReason": null,
+    "LastModifiedTime": "2021-09-29T05:52:26.041Z",
+    "MonitoringScheduleArn": null,
+    "NetworkConfig": null,
+    "ProcessingEndTime": null,
+    "ProcessingInputs": [
+        {
+            "AppManaged": false,
+            "DatasetDefinition": null,
+            "InputName": "mnist_tar",
+            "S3Input": {
+                "LocalPath": "/opt/ml/processing/input",
+                "S3CompressionType": "None",
+                "S3DataDistributionType": "FullyReplicated",
+                "S3DataType": "S3Prefix",
+                "S3InputMode": "File",
+                "S3Uri": "s3://sagemaker-sample-data-us-west-2/algorithms/kmeans/mnist/mnist.pkl.gz"
+            }
+        },
+        {
+            "AppManaged": false,
+            "DatasetDefinition": null,
+            "InputName": "source_code",
+            "S3Input": {
+                "LocalPath": "/opt/ml/processing/code",
+                "S3CompressionType": "None",
+                "S3DataDistributionType": "FullyReplicated",
+                "S3DataType": "S3Prefix",
+                "S3InputMode": "File",
+                "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/kmeans_preprocessing.py"
+            }
+        }
+    ],
+    "ProcessingJobArn": "arn:aws:sagemaker:us-west-2:123456789021:processing-job/unit-testing-processing-jobss",
+    "ProcessingJobName": "unit-testing-processing-jobs",
+    "ProcessingJobStatus": "Stopping",
+    "ProcessingOutputConfig": {
+        "KmsKeyId": null,
+        "Outputs": [
+            {
+                "AppManaged": false,
+                "FeatureStoreOutput": null,
+                "OutputName": "train_data",
+                "S3Output": {
+                    "LocalPath": "/opt/ml/processing/output_train/",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/output/"
+                }
+            },
+            {
+                "AppManaged": false,
+                "FeatureStoreOutput": null,
+                "OutputName": "test_data",
+                "S3Output": {
+                    "LocalPath": "/opt/ml/processing/output_test/",
+                    "S3UploadMode": "EndOfJob",
+                    "S3Uri": "s3://source-data-bucket-592697580195-us-west-2/sagemaker/processing/output/"
```

Comment:
looks like an account id that we should edit out

Reply:
its the marin-public-test-data account, used in other unit tests as well. I think it should be fine?

Reply:
ok, thanks for confirming.